Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1556,6 +1514,56 @@ UpdateFspConfig (
     Fspscfg->PseTsnGbePhyInterfaceType[0]    = 1;
     Fspscfg->PseTsnGbePhyInterfaceType[1]    = 1;
 
+    //TSN_MANUAL_CONFIG
+    if (FeaturePcdGet (PcdTsnConfigEnabled)) {

```

Comment:
I introduced this PCD instead of cfgdata feature so that the boardconfig.py doesn’t need to allocate TSN binaries size for IPFW during build time when it’s off. What is your opinion on this?

Reply:
without the new PCD, you could still try to load TSN components. if load success set the UPDs with actual values.

Reply:
You are right, done.